app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        cmd, source_file, second_file = command.split(maxsplit=3)
+    except ValueError:
+        raise
```

Comment:
Don't raise an exception here. You can print a message here and finish the function execution.